src/blog/2024/05/node-red-mind-stack-with-flowfuse.md

```diff
@@ -0,0 +1,231 @@
+---
+title: The MIND stack with Node-RED and FlowFuse Dashboard 2.0
+subtitle: Ever wanted to deploy the MING stack but want to reduce the complexity?
+description: Ever wanted to deploy the MING stack but want to reduce the complexity?
+date: 2024-05-06
+authors: ["flowfuse-team"]
```

Comment:
You can have 2 authors on a post. Please put yourself in, and the content writer as guest. See the guest directory in the _data directory in the website repo